CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # New Relic Ruby Agent Release Notes #
 
+  ## v6.15.1
```

Comment:
Are we planning on doing a 6.15.1 release? Or should this be 7.0.0?

Reply:
if it would be feasible it would be nice to have the fix sooner :)

Reply:
I wasn't sure where our timeline was falling for 7.0 at the time I wrote 6.15.1.  

7.0.0 release is imminent, so I think we'll skip releasing a 6.15.1